cinema/serializers.py

```diff
@@ -1 +1,71 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("id", "name", "rows", "seats_in_row", "capacity")
+        read_only_fields = ("id",)
```

Comment:
No need to specify id is read-only field in your Serializers, it is read-only by default